packages/pigeon/example/app/ios/Runner/Messages.g.swift

```diff
@@ -222,6 +223,30 @@ class ExampleHostApiSetup {
     } else {
       sendMessageChannel.setMessageHandler(nil)
     }
+    let sendMessageModernAsyncChannel = FlutterBasicMessageChannel(
```

Comment:
can we just reuse the channel?

Reply:
How?

Reply:
can you point me to where this is used? im wondering if you can just use the original channel

Reply:
What do you mean by "original channel"? Each method of Host API got generated `FlutterBasicMessageChannel`.

![image](https://github.com/user-attachments/assets/54f8fc18-c8fa-4bdd-aac9-12f61a258609)

This is default Pigeon generated code flow. 

Maybe I've misunderstood you.

Reply:
Maybe i misunderstood how pigeon is used. Just a strawman - can we share the same channel for both old and new ways of doing things?

Reply:
The way we use channels will not differ at all. We just can not use two handlers for one channel at the same time.

Reply:
Can we have old api to be the wrapper of the new api?

Reply:
Could you provide code sample?

Reply:
nvm, i misunderstood what this code is doing.